aea/cli/scaffold.py

```diff
@@ -55,6 +55,23 @@
 
 
 @click.group()
+@click.option(
+    "-vendor",
+    default="open_aea",
+    type=str,
+    help="Specify component vendor.",
+)
+@click.option(
+    "-package-dir",
+    default="packages/",
+    type=str,
+    help="Path packages directory.",
+)
```

Comment:
Not needed; registry_path is already set on root `aea` command; check `aea.cli.core`